src/Enjin.Platform.Sdk/Enjin.Platform.Sdk/Json/GraphQlParameterJsonConverter.cs

```diff
@@ -32,8 +32,31 @@ public override void Write(Utf8JsonWriter writer, TValue value, JsonSerializerOp
 
         foreach (KeyValuePair<string, object?> variable in value.Parameters)
         {
-            writer.WritePropertyName(variable.Key);
-            writer.WriteRawValue(JsonSerializer.Serialize(variable.Value, options));
+          if (variable.Value != null && variable.Value.GetType().IsGenericType && variable.Value.GetType().GetGenericTypeDefinition() == typeof(List<>)) 
```

Comment:
Would be great if there could be test cases for various scenarios here. Looks like maybe this unit test is meant to cover this functionality and could be expanded?

https://github.com/enjin/platform-csharp-sdk/blob/master/src/Enjin.Platform.Sdk/Enjin.Platform.Sdk.Tests/Unit/Json/GraphQlParameterJsonConverterTest.cs